### DISCUSSION TITLE
Collaboration Julien/Tobias topic

### DISCUSSION THREAD

**Discussion by julien-boudry** [julien-boudry/Condorcet#68] [Collaboration]
@toddy15 

(also #67)

**Comment by toddy15** [comment]
Alright, I've just created a new PR for infection. You might need to mark it as "Draft". There are a few commits after a first (minimal) run of infection, please take a look.

I've just run infection on the file "Candidate.php" and it already spotted 4 escaped mutants. All of those are now fixed -- either in the code or in the tests.
